package.json

```diff
@@ -50,7 +55,7 @@
   "browserslist": [
     "> 1%",
     "last 2 versions",
-    "not ie <= 8"
+    "not ie <= 11"
```

Comment:
`Crypto.getRandomValues(keys);` is not supported by older browsers
https://caniuse.com/getrandomvalues

Reply:
Do we really need IE? It's not supported even by Microsoft already.

Reply:
I don't know.
cc: @kirushik @vas3k

Reply:
This was broken anyway — so I'm OK to drop IE8..10 support.
In general, I'd like BananaSplit to be as conservative in browser support as possible — but only as long as doing so won't introduce any potential attack vectors: say, adding an extra "shim" dependency opens us up for a supply chain attack, and is worse than browser support degradation. (But the actual _feature_ which requires this browser API should be considered very carefully in such case.)

It should all be a reflection of our big design goal to guarantee Banana Split extremely long "shelf life", ideally 10–20 years for the released htmls without any modifications being needed.